docs/articles/developer-info/The-Teams.md

```diff
@@ -0,0 +1,65 @@
+# The Teams in the NUnit project
+
+The NUnit project is made up of several teams, each with its own focus and responsibilities, and with a set of team members.
+
+The team members are a bunch of dedicated people from all over the globe, who are passionate about NUnit and want to make it better. They are all volunteers, and they do this in their spare time.
+
+## The current teams
+
+### Core Team
+
+This team has overall responsibility for the NUnit organization. It defines the vision for NUnit, decides what projects should be included, appoints leads for other teams and maintainers for projects, sets standards and ensures we follow them.
+
+Team Lead: [Rob Prouse](https://github.com/rprouse)
+
+### Framework Team
+
+This team maintains the NUnit framework, which is the assembly referenced by user tests, as well as NUnitLite.
+
+Team Lead: [Rob Prouse](https://github.com/rprouse)
+
+### Engine Team
+
+The Engine Team maintains the NUnit3-Console runner, the Engine and a number of extensions.
+
+Team Lead: Wanted!
+
+### VS Extensions Team
+
+NUnit's extensions to the Visual Studio IDE and dotnet.
+
+Team Lead: [Terje Sandstrom](https://github.com/osiristerje)
+
+### Analyzer Team
+
+The team that works on the NUnit Analyzers Project.
+
+Team Lead: [Mikkel Nylander Bundgaard](https://github.com/mikkelbu)
+
+### Documentation Team
+
+The team that works on the documentation for NUnit.
+
+Team Lead: [Sean Killeen](https://github.com/seankilleen)
+
+## Membership matrix
+
+### Current team memberships, listed in a kind of chronological order
+
+|Name|Core Team|Framework|Engine|VS Extensions|Analyzer|Documentation|
+|--|--|--|--|--|--|--|
+|[Rob Prouse](https://github.com/rprouse) :canada:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|||
+|[Terje Sandstrom](https://github.com/osiristerje) :norway:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|||
+|[Joseph Musser](https://github.com/jnm2) :us:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:|||
+|[Mikkel Nylander Bundgaard](https://github.com/mikkelbu) :denmark:|:heavy_check_mark:|:heavy_check_mark:|:heavy_check_mark:||:heavy_check_mark:||
+|[Sean Killeen](https://github.com/SeanKilleen) :us:|:heavy_check_mark:||||:heavy_check_mark:|:heavy_check_mark:|
+|[Steven Weerdenburg](https://github.com/stevenaw) :canada: |:heavy_check_mark:|:heavy_check_mark:|||||
+|[Ihor Sychevskyi](https://github.com/Arhell) :ukraine:||||||:heavy_check_mark:|
+|[Manfred Brands](https://github.com/manfred-brands) :australia: ||:heavy_check_mark:|||||
```

Comment:
@OsirisTerje I should be listed in the analyzer team

Reply:
Done.   Also approved the Analyzer Team membership.

Reply:
@OsirisTerje I know this has nothing to do with the PR, but given that you have more "power" than I have. Can you also make me the maintainer of the team (currently it is @SeanKilleen as he created the team).

Reply:
Sure!  and Done!